packages/api/src/parse-cli.ts

```diff
@@ -305,16 +305,6 @@ export default function parseCli(argv?: string | readonly string[]) {
         default: [],
         coerce: coerceArr,
       },
-      "record-object-store-id": {
-        describe:
-          "id of the object store that should be used for `record: true` requests that don't otherwise have an os",
-        type: "string",
-      },
-      "supress-record-in-hook": {
-        describe:
-          "do not return record object store in /stream/hook response, even if it is specified in the stream object",
-        type: "boolean",
-      },
```

Comment:
WDYT of keeping these fields as deprecated (e.g. rewrite description explaining it) just so we have a safer deploy? Otherwise we'll have to deploy, but not before we remove the secret, but not before we update livepeer-infra not to use them (safe path executed in the opposite order lol).